src/dao/UserDAO.js

```diff
@@ -26,6 +26,10 @@ class UserDAO extends AbstractContractDAO {
     return storage.contract.then(deployed => deployed.adminCount.call().then(r => r.toNumber()))
   };
 
+  countIUsers () {
```

Comment:
index? we don't need index here, we need to count all users
please rename it to countUsers and decrement resulting value here, not in your action creator